views/models/UserConfigModel/index.js

```diff
@@ -0,0 +1,56 @@
+import colorbrewer from 'colorbrewer'
+
+const UserConfigModel = {
+  configColorOptions: () => {
+    return Object.keys(colorbrewer)
+      .map((key) => colorbrewer[key])
+      .filter((_, i) => i !== 0)
+      .map((palettes) => Object.values(palettes))
+      .flatMap((palette) => palette)
+      .map((palette, i) => ({ label: palette, value: i }))
+  },
+  defaultConfigValues: {
+    category: '',
+    analysis: '',
+    variable: '',
+    label: '',
+    color: {
+      value: 221,
+      label: ['#a6cee3', '#1f78b4', '#b2df8a', '#33a02c', '#fb9a99'],
```

Comment:
Would be nice to store these as descriptive variables.

Reply:
[a290c3c](https://github.com/AMP-SCZ/dpdash/pull/332/commits/a290c3c5c19a3945455d477227dd4c30e793bb71)